tests/test_batoid_wcs.py

```diff
@@ -597,8 +598,46 @@ def test_config():
     np.testing.assert_raises(NotImplementedError,
                              galsim.config.BuildWCS, config['image'], 'wcs', config)
 
+def test_focal_to_pixel() -> None:
+    det = imsim.get_camera()[0]
+    x = np.linspace(start=-190.615, stop=-188.615, num=4)
+    y = np.linspace(start=-317.255, stop=-315.255, num=4)
+    x_t, y_t = batoid_wcs.focal_to_pixel(
+        np.tile(x, len(y)), np.repeat(y, len(x)), det
+    )
+
+    x_expected = y_expected = np.linspace(start=-100.0, stop=100.0, num=4)
+    np.testing.assert_almost_equal(x_t, np.tile(x_expected, len(y)))
+    np.testing.assert_almost_equal(y_t, np.repeat(y_expected, len(x)))
+
+
+def test_pixel_to_focal() -> None:
+    det = imsim.get_camera()[0]
+    x = y = np.linspace(start=-100.0, stop=100.0, num=4)
+    x_t, y_t = batoid_wcs.pixel_to_focal(
+        np.tile(x, len(y)), np.repeat(y, len(x)), det
+    )
+
+    x_expected = np.linspace(start=-190.615, stop=-188.615, num=4)
+    y_expected = np.linspace(start=-317.255, stop=-315.255, num=4)
+    np.testing.assert_almost_equal(x_t, np.tile(x_expected, len(y)))
+    np.testing.assert_almost_equal(y_t, np.repeat(y_expected, len(x)))
+
+
 if __name__ == "__main__":
-    test_wcs_fit()
-    test_imsim()
-    test_intermediate_coord_sys()
-    test_config()
+    import unittest
+
+    class TestLoader(unittest.TestLoader):
+        """unittest TestLoader to load test functions."""
+
+        def loadTestsFromModule(self, module, *args, pattern=None, **kws):
+            tests = super().loadTestsFromModule(module, *args, pattern=pattern, **kws)
+            test_functions = (
+                f
+                for f_name, f in globals().items()
+                if f_name.startswith("test_") and callable(f)
+            )
+            tests.addTests([unittest.FunctionTestCase(f) for f in test_functions])
+            return tests
+
+    unittest.main(testLoader=TestLoader())
```

Comment:
Why all this unittest boilerplate?  I believe this is functionally equivalent to:
```
test_functions = (
    f 
    for f_name, f in globals().items() 
    if f_name.startswith("test_") and callable(f))
for test_fn in test_functions:
    test_fn()
```
which is simpler and also doesn't activate my distaste for the unittest module...  :)

Reply:
I just wanted to benefit from `unittest` (it prints what failed, what it runs, ...). And all the rest of the test files in imsim also are using the module.
But of course, we also can use a different test loader. I dont care

Reply:
The boilerplate code is necessary to let the `unittest` module discover functions.
I really understand your distaste for `unittest` just because of the fact that this is not the default of `unittest`.
But this test file is the only one in imsim which does not pack the tests in `unittest.TestSuite` classes. We have to do this manually to mimic the behaviour of pytest

Reply:
For now, I moved as much boilerplate code as I could to `tests.__init__` and use it in `test_batoid_wcs` and `test_utils`.